src/components/OfferMyAccountCard/OfferMyAccountCard.tsx

```diff
@@ -174,7 +193,18 @@ const OfferMyAccountCard = ({ offerId }: OfferMyAccountCardProps) => {
     }
   };
 
+  const getPendingUpgradeCopy = () => {
+    return t(
+      `offer-card.switch-details.upgrade-pending`,
+      `Your upgrade is pending.`
+    );
+  };
+
   const getDescription = () => {
+    if (isUpgradePending) {
+      return getPendingUpgradeCopy();
```

Comment:
i think it doesn't need to be a separate function, you can just return the `t` call here directly